v20.1/admin-ui-hardware-dashboard.md

```diff
@@ -86,9 +88,18 @@ For Mac OS, this graph is not populated and shows zero disk IOPS in progress. Th
 
 <img src="{{ 'images/v20.1/admin_ui_available_disk_capacity.png' | relative_url }}" alt="CockroachDB Admin UI Disk Capacity graph" style="border:1px solid #eee;max-width:100%" />
 
-- In the node view, the graph shows the available storage capacity for the selected node.
+Metric | Description
+--------|--------
+**Available Disk Capacity** | Free disk space available to CockroachDB data on each node.
+
+### Capacity metrics
+
+The **available** disk capacity equals the amount of empty disk space, up to the value of the maximum store size. The store size is determined as follows:
+
+- If a store size was specified using the [`--store`](cockroach-start.html#store) flag when starting nodes, this value is used as the limit for CockroachDB data.
+- If no store size has been explicitly set, the actual disk capacity is used as the limit for CockroachDB data.
 
-- In the cluster view, the graph shows the available storage capacity across all nodes in the cluster.
+This disk space is not used by the Cockroach binary, operating system, and other system files.
```

Comment:
I'm a little confused.  Do you mean that the disk space set aside for the store is not used by the `cockroach` binary?  If so, what happens if I do not set a store size?

It seems like this sentence is an answer to a question.  What was the question?